cross-lib/run_enki_on_all.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+while [ -n "$1" ]; do
+  case "$1" in
+    --command) 
+      shift; arg_command=$1
+    ;;
+    *)
+      echo "Invalid argument $1"
+      exit 1
+    ;;
+  esac
+  shift
+done
+
+[[ $arg_command ]] || ( echo "ERROR: A command was not provided. Typical examples are 'all-git-pdf' or 'all-git-web' or 'all-git-epub'" && exit 1 )
+
+all_books="cross-lib/book_data/AUTO_books.txt"
+test -f $all_books || ( echo "ERROR: Book list not found at ${all_books}" && exit 1 )
+
+mkdir -p cross-lib/logs/
+
+# https://stackoverflow.com/a/20983251
+echo_green() { echo -e "$(tput setaf 2)$*$(tput sgr0)"; }
+echo_red() { echo -e "$(tput setaf 1)$*$(tput sgr0)"; }
+
+run_and_log_enki () {
+  echo "  running command $1 on $2 $3"
+  start_time=$(date +%s)
```

Comment:
I forgot to mention this:
```bash
  data_dir="./data/$3-$1"
  [[ -d "$data_dir" ]] || mkdir -p "$data_dir"
  start_time=$(date +%s)
  ../enki --data-dir "$data_dir" --command "$1" --repo "openstax/$2" --book-slug "$3" --style default --ref main
```
Enk is up one directory and the data dirs were not being made.

Reply:
ah - i have been testing this from enki root, i.e. running `./cross-lib/run_enki_on_all.sh --command all-git-web`. other paths also make this assumption.